src/pages/OpprettPameldingPage.tsx

```diff
@@ -2,20 +2,62 @@ import { PameldingResponse } from '../api/data/pamelding.ts'
 
 import { PameldingHeader } from '../components/pamelding/PameldingHeader.tsx'
 import { PameldingForm } from '../components/pamelding/PameldingForm.tsx'
+import { PameldingFormValues } from '../model/PameldingFormValues.ts'
+import { useDeferredFetch } from '../hooks/useDeferredFetch.ts'
+import { oppdaterKladd } from '../api/api.ts'
+import { useAppContext } from '../AppContext.tsx'
+import { KladdRequest } from '../api/data/kladd-request.ts'
+import { generateMalFromResponse } from '../utils/pamelding-form-utils.ts'
+import { useState } from 'react'
 
 export interface OpprettPameldingPageProps {
   pamelding: PameldingResponse
 }
 
 export const OpprettPameldingPage = ({ pamelding }: OpprettPameldingPageProps) => {
+  const { enhetId } = useAppContext()
```

Comment:
Må dette håndteres her? kunne vi ikke hatt dette i den komponenten som viser lagrestaten? så slipper vi å sende så mye herfra? den komponenten kan vi også si at bare skal rendres for kladd, og ha den useEffect-en inni den?

Reply:
Det kan være en lur løsning, flytte alt inn i "PameldingLagringsstatus"... La til den til slutt så tenkte ikke over at det er en mulighet